### PR TITLE
fix broken courses link on /congrats

### PR DESCRIPTION
**Pull request by Erin007**
There was a broken link on /congrats with the view course catalog button. 
screen shot 2017-11-29 at 11 58 26 am

This fixes that link.

### CODE REVIEW

File: apps/src/templates/congratsBeyondHocActivityCards.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const CSFAccelerated = {
 const CourseCatalog = {
   title: i18n.courseCatalogTitle(),
   description: i18n.courseCatalogDescription(),
-  link: "/courses/view?=teacher",
```
Comment (breville) [inline apps/src/templates/congratsBeyondHocActivityCards.js:23]: This should have been `/courses?view=teacher` fwiw. Was the intent to show the teacher version of /courses?  if you're clicking on this when signed out and want a teacher view, you need this parameter.

**Comment by breville** [comment]
Please test the whole matrix of clicking on this (signed out, signed in student, signed in teacher)

**Comment by Erin007** [comment]
@tanyaparker If we know the user is a signed in student do we still want them to see the teacher version of /courses?

File: apps/src/templates/congratsBeyondHocActivityCards.js (hunk 1 above)
Comment (tanyaparker) [reply apps/src/templates/congratsBeyondHocActivityCards.js:23]: For non-EN it doesn't matter (they get the same). So I prefer the simple `/courses` without the teacher param.

**Comment by tanyaparker** [comment]
@Erin007 when does a signed in student still see the teacher version of courses? with this change that should never happen right?

**Comment by Erin007** [comment]
When the redirect is to /courses (like it is now), then courses will handle whether you see the teacher/student version.  Prior to this change, we were attempting to redirect everyone non-EN including students to the teacher version of courses.

**Pull request merged**